src/Validator.php

```diff
@@ -29,18 +29,26 @@ class Validator
      *          ['lengthBetween', 4, 10, 'message'=>'test 2'],
      *      ],
      *  ];.
+     *
+     * @var array<string,array<string|array<string|int|callable>>>
      */
     public array $rules = [];
 
     /**
      * Array of conditional rules in following format:
      *  [
-     *      [$conditions, $then_rules, $else_rules],
+     *      [
+     *          $conditions, // array of conditions
+     *          $then_rules, // array in $this->rules format which will be used if conditions are met
+     *          $else_rules  // array in $this->rules format which will be used if conditions are not met
+     *      ],
      *  ].
      *
-     * $conditions - array of conditions
-     * $then_rules - array in $this->rules format which will be used if conditions are met
-     * $else_rules - array in $this->rules format which will be used if conditions are not met
+     * @var array{
```

Comment:
same here and put it on one line

Reply:
I just tested locally - phpstan deal with multi lines just fine.